docs/docs/cmd/spo/tenant/tenant-settings-set.mdx

```diff
@@ -265,6 +424,84 @@ m365 spo tenant settings set [options]
+
+`--DisableBackToClassic [DisableBackToClassic]`
+: Disables or enables the back to classic link for libraries and lists. Allowed values `true`, `false`.
+
+`--DisablePersonalListCreation [DisablePersonalListCreation]`
+: Allows creating personal lists within the OneDrive for Business site of the user. Allowed values `true`, `false`.
+
+`--ViewInFileExplorerEnabled [ViewInFileExplorerEnabled]`
+: Enables or disables the ability to use View in Explorer in Microsoft Edge. Allowed values `true`, `false`.
+
+`--AllowGuestUserShareToUsersNotInSiteCollection [AllowGuestUserShareToUsersNotInSiteCollection]`
+: Allows guest users to share to users not in the site collection. Allowed values `true`, `false`.
+
+`--BlockSendLabelMismatchEmail [BlockSendLabelMismatchEmail]`
+: Allows blocking of the automated e-mail being sent when somebody uploads a document to a site that's protected with a sensitivity label and their document has a higher priority sensitivity label than the sensitivity label applied to the site. Allowed values `true`, `false`.
+
+`--CoreDefaultLinkToExistingAccess [CoreDefaultLinkToExistingAccess]`
+: Sets default share link to existing access on core partition. Allowed values `true`, `false`.
+
+`--CoreRequestFilesLinkEnabled [CoreRequestFilesLinkEnabled]`
+: Enables or disables the Request files link for all SharePoint sites (not including OneDrive sites). If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.
+
+`--OneDriveRequestFilesLinkEnabled [OneDriveRequestFilesLinkEnabled]`
+: Enables or disables the Request files link on the OneDrive partition for all OneDrive sites. If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.
+
+`--DisableDocumentLibraryDefaultLabeling [DisableDocumentLibraryDefaultLabeling]`
+: Allows tenant admins to disable the capability of configuring a default sensitivity label for a document library. Allowed values `true`, `false`.
+
+`--DisableVivaConnectionsAnalytics [DisableVivaConnectionsAnalytics]`
+: Specifies whether Viva Connections Analytics is disabled or enabled on the tenant. Allowed values `true`, `false`.
+
+`--DisplayNamesOfFileViewersInSpo [DisplayNamesOfFileViewersInSpo]`
+: Allows configuring whether display name of people who view the file are visible in the property pane of the site in SharePoint site collection. Allowed values `true`, `false`.
+
+`--EnableAIPIntegration [EnableAIPIntegration]`
+:  Specifies whether Azure Information Protection (AIP) should be enabled on the tenant. Allowed values `true`, `false`.
+
+`--EnableRestrictedAccessControl [EnableRestrictedAccessControl]`
+: Specifies whether the policy which determines Restricted Access Control Policy is enabled. Allowed values `true`, `false`.
+
+`--ExternalUserExpirationRequired [ExternalUserExpirationRequired]`
```

Comment:
Can we place this option next to the option that configures how many days an expiration time is? This will make sure that related options are grouped together in the docs.

Reply:
Yes, will push the new commit for this suggestion.